server/game/cards/09.1-FtE/PurityOfSpirit.js

```diff
@@ -0,0 +1,30 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+const { CardTypes } = require('../../Constants');
+
+class PurityOfSpirit extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Choose a bushi character to honor',
+            condition: () => this.game.isDuringConflict(),
+            target: {
+                cardType: CardTypes.Character,
+                cardCondition: card => card.hasTrait('bushi') && card.isParticipating(),
+                gameAction: AbilityDsl.actions.multiple([
+                    AbilityDsl.actions.honor(),
+                    AbilityDsl.actions.delayedEffect({
+                        when : {
+                            onConflictFinished: () => true
+                        },
+                        gameAction: AbilityDsl.actions.discardStatusToken(context => ({ target: context.target.personalHonor })),
+                        message: '{2} removes the {3} from {1} due to the delayed effect of {0}',
+                        messageArgs: context => [context.player, context.target.personalHonor]
```

Comment:
Have you checked to make sure this works in chat?  I don't think we've ever sent a token direct to chat before....

Reply:
It's in the tests:
```
expect(this.getChatLogs(5)).toContain('player1 removes the Honored Token from Shiba Tsukune due to the delayed effect of Purity of Spirit');
```
Are we bothered about the capitalisation on "Honored Token"?